elixir_phoenix/web/controllers/client_controller.ex

```diff
@@ -0,0 +1,58 @@
+defmodule ElixirPhoenix.ClientController do
+  use ElixirPhoenix.Web, :controller
+
+  alias EboshiApiShootoutElixirPhoenix.Client
+  alias EboshiApiShootoutElixirPhoenix.Repo
+
+  plug :scrub_params, "client" when action in [:create, :update]
+
+  def index(conn, _params) do
+    clients = Repo.all(Client)
+    render(conn, "index.json", clients: clients)
+  end
+
+  def create(conn, %{"client" => client_params}) do
```

Comment:
Just out of curiosity, how much of this was generated scaffold code, and how much did you have to suss out on your own?